DataFormats/SiStripClusterSoA/src/classes_def.xml

```diff
@@ -0,0 +1,6 @@
+<lcgdict>
+  <class name="edm::StdArray<unsigned char,768>"/>
```

Comment:
This dictionary should go in `DataFormats/Common/src/classes_def.xml`

Reply:
I'm not sure why you mention "ISO-like" or "lcg".

There are unit tests for the overall implementation of SoA templates and PortableCollections under various packages, like `DataFormats/SoATemplate/test/`, `DataFormats/Portable/test/`, `DataFormats/PortableTestObjects/test/`, and `HeterogeneousCore/AlpakaTest/test/`.

If you would like to write a unit test specifically for the types under `DataFormats/SiStripClusterSoA/`, you can add them under `DataFormats/SiStripClusterSoA/src/test/`.